mediacapture-streams/stream-api/mediastreamtrack/mediastreamtrack-end.html

```diff
@@ -29,7 +29,7 @@ <h1 class="instructions">Description</h1>
     vidTrack.onended = t.step_func(function () {
         assert_equals(vidTrack.readyState, "ended", "Video track has been ended as expected");
         assert_equals(audTrack.readyState, "ended", "Audio track has been ended as expected");
-        assert_true(stream.ended, "MediaStream has been ended as expected");
+        assert_false(stream.active, "MediaStream has been inactive as expected");
```

Comment:
Please also update the description at line 15, s/correctly set into ended state/inactive/, thanks.